src/FastRouteRouter.php

```diff
@@ -444,10 +435,6 @@ private function injectRoute(Route $route) : void
             $methods = self::HTTP_METHODS_STANDARD;
         }
 
-        if (empty($methods)) {
-            $methods = self::HTTP_METHODS_EMPTY;
```

Comment:
We shouldn't add default empty methods if any method is not allowed.

I think it is better to throw an exception when we try add route without any methods.